mods/cnc/maps/cnc64gdi01/cnc64gdi01.lua

```diff
@@ -0,0 +1,209 @@
+CommandoReinforcements = { "rmbo" }
+MCVReinforcements = { "mcv" }
+
+inf1 = { "e4" }
+auto1 = { "stnk", "stnk" }
```

Comment:
I'd suggest using
```
AutocreateSquads =
{
	{ "stnk", "stnk" },
	{ "ftnk", "ftnk" },
	{ "ltnk", "ltnk", "bike" },
	{ "arty", "arty", "bike", "bike" },
	{ "ltnk", "ltnk" },
	{ "stnk", "stnk" },
	{ "ltnk", "ltnk" },
	{ "arty", "arty" }
}
```
It should work the same way and we avoid this `autoX` stuff.